policyengine_us/parameters/gov/hhs/chip/child/min_age.yaml

```
@@ -0,0 +1,9 @@
+description: The US Government set this as the minimum age for children to qualify for CHIP.
```

Although we should just remove
```suggestion
description: The US limits the Children's Health Insurance Program to children this age or older.
```